etc/profile-a-l/email-common.profile

```diff
@@ -8,6 +8,7 @@ include email-common.local
 #include globals.local
 
 noblacklist ${HOME}/.bogofilter
+noblacklist ${HOME}/.bsfilter
```

Comment:
A `noblacklist foo` only makes sense when there's a `blacklist foo`, like in the ${HOME}/.bogofilter case:

https://github.com/netblue30/firejail/blob/1f5f84f9596ba7d26c2f2e35dad7728e12afaec9/etc/inc/disable-programs.inc#L53

This protects the configuration from apps that don' need access to it.

Please add the needed `blacklist ${HOME}/.bsfilter` to `disable-programs.inc` (respecting sort order).